Doc/library/_interpreters.rst

```diff
@@ -0,0 +1,90 @@
+:mod:`_interpreters` --- Low-level interpreters API
+===================================================
+
+.. module:: _interpreters
+   :synopsis: Low-level interpreters API.
+
+.. versionadded:: 3,7
+
+--------------
+
+This module provides low-level primitives for working with multiple
+Python interpreters in the same runtime in the current process.
+
+More information about (sub)interpreters is found at
+:ref:`sub-interpreter-support`, including what data is shared between
+interpreters and what is unique.  Note particularly that interpreters
+aren't inherently threaded, even though they track and manage Python
+threads.  To run code in an interpreter in a different OS thread, call
+:func:`run_string` in a function that you run in a new Python thread.
```

Comment:
Can we call the function `run` as per the PEP?

Reply:
Note that the PEP is all about the high-level API we want to expose to users.  In contrast, `run_string()` is the low-level function focused strictly as an analog to `PyRun_String*()`.  As we support other runnable things (e.g. code objects, functions), we will add corresponding low-level functions.  The eventual high-level method (i.e. `Interpreter.run()` will call the appropriate low-level function.

Also, I'm removing the docs for now.